packages/discovery-provider/plugins/pedalboard/apps/backfill-audio-analyses/.eslintrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  root: true,
+  extends: ["custom-server"],
```

Comment:
Just a thought, should `custom-server` extend `audius`? maybe an @alecsavvy q

Reply:
oh yeah it should